hooks/useIsUserLockedOut.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { useEffect, useState } from 'react';
+
+/**
+ * Function to constantly check the current time every hour and set the lockout accordingly.
+ * @returns - The value of lockedOut.
+ */
+const useIsUserLockedOut = (): boolean => {
+  const [lockedOut, setLockedOut] = useState<boolean>(false);
+  useEffect(() => {
+    /**
+     * Checks if the user is locked out from making a pick.
+     */
+    const checkLockout = (): void => {
+      const currentDateAndTime = new Date();
+      const day = currentDateAndTime.getUTCDay();
+      const hours = currentDateAndTime.getUTCHours();
+      if (
+        (day === 5 && hours >= 0) || // Thursday at 9pm EST
+        day > 5 || // Friday and Saturday
+        day === 0 || // Sunday
+        day === 1 || // Monday
+        (day === 2 && hours < 12) // Tuesday at 9am EST
+      ) {
+        setLockedOut(true);
+      } else {
+        setLockedOut(false);
+      }
+    };
+
+    checkLockout();
+
+    const intervalId = setInterval(checkLockout, 60 * 60 * 1000);
```

Comment:
Again, what is 60, 60, and 1000? Put these into COST instead, so anyone can read and understand what each value means. Also, what's the reasoning behind useing `setInterval`?

Reply:
`60 * 60 * 1000` creates a `setInterval` that checks the time once per hour. The reasoning behind the `setInterval` is so that it checks the time every hour and can appropriately update the state of `setLockedOut`.

What do you mean by COST?

Reply:
It was a long week. I meant CONST. A constant variable. What represents these individual numbers?